clickhouse/base/socket.cpp

```diff
@@ -72,6 +72,18 @@ void SetNonBlock(SOCKET fd, bool value) {
 #endif
 }
 
+void SetTimeout(SOCKET fd, SocketTimeoutParams params) {
+#if defined(_unix_)
```

Comment:
Shoud we add the same for windows system?

Reply:
If you can validate build on windows os.

Reply:
oh, sorry, i have no opportunity to validate build on windows os.

Reply:
OK.